src/memtable.rs

```diff
@@ -1176,9 +1213,64 @@ impl<A: AllocatorTrait> MemTableRecoverContext<A> {
 
     pub fn merge_append_context(&self, append: MemTableRecoverContext<A>) {
         self.memtables
-            .apply_append_writes(append.log_batch.clone().drain());
+            .apply_append_writes(append.tombstone_items.into_iter());
         self.memtables.merge_append_table(append.memtables);
     }
+
+    #[inline]
+    fn is_tombstone(item: &LogItem) -> bool {
```

Comment:
What does `is_tombstone` mean?

Reply:
Deletion marks like Compact or Clean. They need to be passed down to older files to fully delete history data.